doc/nrf-udev.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+RULESFILE_NRF="/etc/udev/rules.d/71-nrf.rules"
```

Comment:
Any reason for choosing the number 71? Is there a convention for these numbers? Just curious.

Reply:
It needs to be between default udev rules at level 50 and modemmanager 77. I don't know about other conventions.